www/class/CentreonLDAPAdmin.class.php

```diff
@@ -599,14 +585,16 @@ public function deleteConfiguration($configList = [])
     public function setStatus($status, $configList = array())
     {
         if (count($configList)) {
-            $statement = $this->db->prepare(
-                "UPDATE auth_ressource
-                SET ar_enable = :ar_enable
-                WHERE ar_id IN (:configList)"
-            );
-            $statement->bindValue(':ar_enable', $status, \PDO::PARAM_INT);
-            $statement->bindValue(':configList', implode(',', $configList), \PDO::PARAM_STR);
-            $statement->execute();
+            foreach ($configList as $val) {
```

Comment:
are you sure that it is not the key which is used ?

Reply:
reverted